src/firejail/fs_etc.c

```diff
@@ -76,6 +76,44 @@ void fs_machineid(void) {
 	}
 }
 
+// Duplicate directory structure from src to dst by creating empty directories.
+// The paths _must_ be identical after their respective prefixes.
+// When finished, dst will point to the target directory. That is, if
+// it starts out pointing to a file, it will instead be truncated so
+// that it contains the parent directory instead.
+static void build_dirs(char *src, char *dst, size_t src_prefix_len, size_t dst_prefix_len) {
+    char *p = src + src_prefix_len + 1;
```

Comment:
This file uses tabs for indentation.

Reply:
Whoops, completely missed that. Thanks for the head up.

Reply:
> Whoops, completely missed that. Thanks for the heads up.

No problem.  So could you fix the indentation, amend the commit and force-push?

I'd suggest a command to do it automatically, but the proposed `build_dirs`
function is indented using 4 spaces and the changes in `duplicate` are indented
with a mix of 4 and 8 spaces.

Reply:
> Sure! I just pushed the update.
>
> Ah, yes, I even mixed tab length with my spaces, so I should've noticed that
> inconsistency :)

Thanks.

> Thanks for reviewing. As I wrote above, I'm not really happy with this one,
> so I'm all ears for suggestions for improvements. Don't have anything against
> rewriting it, perhaps using sprintf's for building up paths instead.

No problem.  I'm not really familiar with firejail's C code base, so based on
the output of `tig --no-merges master -- '*.h' '*.c'`, I think someone like
@smitsohu would be of more help in that regard.